util.py

```diff
@@ -96,8 +96,14 @@ def sql_alchemy_to_dict(
     sql_alchemy_object: Any,
     for_frontend: bool = False,
     column_whitelist: Optional[Iterable[str]] = None,
+    column_blacklist: Optional[Iterable[str]] = None,
+    column_rename_map: Optional[Dict[str, str]] = None,
 ):
-    result = __sql_alchemy_to_dict(sql_alchemy_object, column_whitelist)
+    result = __sql_alchemy_to_dict(
+        sql_alchemy_object, column_whitelist, column_blacklist
+    )
+    if column_rename_map:
+        result = {column_rename_map.get(k, k): v for k, v in result.items()}
```

Comment:
only works for dict like sql_alchemy_object changes. Could also be a list of dicts (e.g. an all request result)